### ISSUE TITLE
EIP-4895: Beacon chain push withdrawals as operations

### ISSUE THREAD

**Issue by ChihChengLiang** [privacy-scaling-explorations/zkevm-circuits#1369]
### Describe the feature you would like

[EIP-4895: Beacon chain push withdrawals as operations](https://eips.ethereum.org/EIPS/eip-4895)

### Additional context

Pending analysis of what work should be done.

**Comment by ChihChengLiang** [comment]
This task requires modifications in contract creation transaction, CREATE, and CREATE2. Better do it after #1130 is resolved.

**Comment by KimiWu123** [comment]
Since this is a large task, I'll split it into 3 part
1. pi circuit integration - https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1634
2. withdrawal circuit
3. super circuit and other remaining integration
